components/formats-api/src/loci/formats/IFormatReader.java

```diff
@@ -354,6 +354,22 @@ Object openPlane(int no, int x, int y, int w, int h)
    */
   boolean isGroupFiles();
 
+  /**
+   * Set the fill value for undefined pixels.
+   * This can be used to override the default fill value
+   * defined in a reader.
+   *
+   * @param color value that will be used to fill pixel byte arrays
+   */
+  void setFillColor(Byte color);
```

Comment:
Do these new methods to the `IFormatReader` interface constitute backward incompatible changes since third-party reader which does not extend `FormatReader` will need to be updated to implement them?

Reply:
Yes. Anything that extends `FormatReader` (or any other existing reader implementation) shouldn't require changes, but just implementing `IFormatReader` means that `setFillColor(Byte)` will need to be implemented as part of updating to 6.13.0.